common/utils/src/main/resources/error/error-conditions.json

```
@@ -572,6 +572,12 @@
     ],
     "sqlState" : "42703"
   },
+  "COLUMN_VALUE_IS_NULL" : {
```

Let's name in more precise way:
```suggestion
  "ROW_VALUE_IS_NULL" : {
```

---

Updated.